.github/workflows/buildcache.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     needs: check_secrets
     if: ${{ needs.check_secrets.outputs.secrets_available == 'true' }}
     strategy:
+      max-parallel: 2
```

Comment:
This is unrelated to terminating duplicate pushes. It limits the concurrency within the PR matrix itself to two rather than whatever resources are available.

Reply:
The idea was that seeding the build cache is not super-important, so I reduced the concurrency in the hope it would allow more workers for other jobs. For now, I remove `max-parallel` from the PR.